src/angularComponent/ngcOmniboxController.js

```diff
@@ -0,0 +1,11 @@
+import OmniboxController from 'src/coreComponent/OmniboxController.js';
```

Comment:
Core to me is a set of objects and/or functions that can be imported and called **from** the controller. I imagine that some of these functions probably need to be passed DOM elements to work. Since the controller has access to DOM via `this.$element`, we can't use controller sub-classing, we have to just do things like:

```
class Controller {
  $postLink() {
    doSomeCoreSetupBehavior(this.$element[0]);
  }
}
```